requirements.txt

```diff
@@ -6,5 +6,5 @@ scikit-learn==0.22.1
 pandas==0.25.3
 seaborn==0.10.0
 loguru==0.4.1
-tensorflow==2.1.0
-ruamel.yaml~=0.16.10
\ No newline at end of file
+tensorflow==2.2.0
+ruamel.yaml~=0.16.10
```

Comment:
Using conda I had this change on line 10
```suggestion
ruamel.yaml==0.16.10
```

Reply:
Did you try with an updated version of conda @Thomas-Galtier ?

Reply:
Yes but I already had an up-to-date version of conda, so I still got a error.